packages/editor/src/toolbar/tools/utils.tsx

```diff
@@ -38,3 +40,132 @@ export function menuButtonToTool(
     );
   };
 }
+
+/**
+ * Inserts a paragraph at the end of selection.
+ */
+function insertParagraphAtTheEndOfSelection(editor: Editor, tr: Transaction) {
+  if (!editor.current) return;
+  const docSize = editor.current.state.doc.content.size;
+  let selectionEnd = tr.selection.to + 1;
+  selectionEnd = docSize < selectionEnd ? docSize - 1 : selectionEnd;
```

Comment:
This can be in the same like and as a `const`

Reply:
This will require writing `tr.selection.to + 1` twice and I want to avoid doing that.